addons/mail/static/src/messaging/component/activity/activity_tests.js

```diff
@@ -781,6 +781,103 @@ QUnit.test('activity click on edit', async function (assert) {
     );
 });
 
+QUnit.test('activity edition', async function (assert) {
+    assert.expect(15);
+
+    await this.start({
+        async mockRPC(route, args) {
+            if (route === '/web/dataset/call_kw/mail.activity/activity_format') {
+                assert.step('activity_format');
+                return [{
+                    id: 12,
+                    can_write: true,
+                    icon: 'fa-check',
+                    res_id: 42,
+                    res_model: 'res.partner',
+                }];
+            }
+            return this._super(...arguments);
+        },
+        intercepts: {
+            do_action: function (ev) {
```

Comment:
shorthand

Note: I just realized I did the same mistake on my current PR by copying it from elsewhere